djongo/sql2mongo/sql_tokens.py

```diff
@@ -40,9 +42,14 @@ def tokens2sql(token: Token,
                 except ValueError:
                     yield SQLIdentifier(token[0][1], query)
                 else:
-                    yield SQLConstIdentifier(token, query)
+                    # TMiles-2022.XI.18 - Specify int identifier to match original functionality.
+                    yield SQLConstIntIdentifier(token, query)
             elif isinstance(token[0], Function):
                 yield SQLFunc.token2sql(token, query)
+            elif token[0].ttype == tokens.Name.Placeholder:
+                # TMiles-2022.XI.18
+                # Use a parameterized const identifier to handle newer django foreign key lookup queries.
+                yield SQLConstParameterizedIdentifier(token, query);
```

Comment:
Added extra semicolon at the end.

Reply:
Why?  Python, as far as I know doesn't require colons and the existing code base doesn't include semicolons for line endings.  Why add it to this one line?  (And looking back at my commit, I did use one on another line, but that's because I'm from a c#/javascript/etc, background and type semicolons on the ends of lines by habit, and it shouldn't be there either)

Reply:
Sorry. I mean to remove it from the line.

Reply:
Done.  Sorry it's taken me so long.